pkg/config/deployment_cloudrun.go

```diff
@@ -47,5 +47,5 @@ type CloudRunSyncStageOptions struct {
 // CloudRunPromoteStageOptions contains all configurable values for a CLOUDRUN_PROMOTE stage.
 type CloudRunPromoteStageOptions struct {
 	// Percentage of traffic should be routed to the new version.
-	Percent int `json:"percent"`
+	Percent Percentage `json:"percent"`
```

Comment:
Oops, the field name and tag should be `percentage`.

Reply:
Both `Percent` and `Percentage` is lgtm but change to `Percentage` for now is a breaking change so I suggest not doing that 😄

Reply:
Yes, I know. We should not change it. I just wanted to say that I made a mistake on them. Percent is often used to express from 0 to 1, but percentage is from 0 to 100.

Reply:
I see, that diff is a dangerous thing 😱